src/main/kotlin/apply/ui/admin/selections/SelectionView.kt

```diff
@@ -106,6 +134,79 @@ class SelectionView(
         }
     }
 
+    private fun createEvaluationTargetsGrid(evaluationTargets: List<EvaluationTargetResponse>): Component {
+        return Grid<EvaluationTargetResponse>(10).apply {
+            addSortableColumn("이름", EvaluationTargetResponse::name)
+            addSortableColumn("이메일", EvaluationTargetResponse::email)
+            addSortableColumn("합계", EvaluationTargetResponse::totalScore)
+            addSortableColumn("평가 상태", EvaluationTargetResponse::evaluationStatus)
+            addSortableColumn("평가자", EvaluationTargetResponse::administratorId)
+            addColumn(createEvaluationButtonRenderer()).apply { isAutoWidth = true }
+            setItems(evaluationTargets)
+        }
+    }
+
+    private fun createEvaluationButtonRenderer(): Renderer<EvaluationTargetResponse> {
+        return ComponentRenderer<Component, EvaluationTargetResponse> { response ->
+            createPrimarySmallButton("평가하기") {
+                EvaluationTargetFormDialog(evaluationTargetService, response.id) {
+                    selectedTabIndex = tabs.selectedIndex
+                    removeAll()
+                    add(
+                        createTitle(),
+                        createContent()
+                    )
+                }
+            }
+        }
+    }
+
+    private fun createTabComponents(tabsToPages: LinkedHashMap<Tab, Component>): Pair<Tabs, Div> {
+        val tabs = Tabs()
+        tabs.add(*(tabsToPages.keys).toTypedArray())
+        val pages = Div(*tabsToPages.values.toTypedArray())
+        pages.apply { setWidthFull() }
+        tabsToPages.values.forEach { it.isVisible = false }
+
+        tabs.addSelectedChangeListener {
+            tabsToPages.values.forEach { it.isVisible = false }
+            val selectedPage = tabsToPages[tabs.selectedTab]
+            selectedPage?.isVisible = true
+        }
+
+        val selectedPage = tabsToPages[tabs.selectedTab]
+        selectedPage?.isVisible = true
+        return Pair(tabs, pages)
```

Comment:
```suggestion
        val selectedPage = tabsToPages[tabs.selectedTab]?.apply { isVisible = true }
        return tabs to pages
```
이런 느낌?

Reply:
```suggestion
        tabsToPages[tabs.selectedTab]?.apply { isVisible = true }
        return tabs to pages
```

보다 깔끔..!
범은 수신 객체 지정 람다를 잘 활용하는 거 같아요!!

배워갑니다😀